src/bci_build/package/basecontainers.py

```diff
@@ -330,6 +334,57 @@ def _get_minimal_kwargs(os_version: OsVersion):
     )
 
 
+# SL Micro 6.0 GA kernel container devel
+_SLM60_BASEURL = "https://api.opensuse.org/public/build/SUSE:ALP:Source:Standard:Core:1.0:Build/standard/"
+_SLM60_KERNEL_PACKAGES = [
+    "x86_64/patchinfo.ga/kernel-devel-6.4.0-17.1.noarch.rpm",
+    "x86_64/patchinfo.ga/kernel-macros-6.4.0-17.1.noarch.rpm",
+    "x86_64/patchinfo.ga/kernel-syms-6.4.0-17.1.x86_64.rpm",
+    "x86_64/patchinfo.ga/kernel-default-devel-6.4.0-17.1.x86_64.rpm",
+    "aarch64/patchinfo.ga/kernel-syms-6.4.0-17.1.aarch64.rpm",
+    "aarch64/patchinfo.ga/kernel-default-devel-6.4.0-17.1.aarch64.rpm",
+    "aarch64/patchinfo.ga/kernel-64kb-devel-6.4.0-17.1.aarch64.rpm",
+    "s390x/patchinfo.ga/kernel-syms-6.4.0-17.1.s390x.rpm",
+    "s390x/patchinfo.ga/kernel-default-devel-6.4.0-17.1.s390x.rpm",
+]
+_SLFO_KEY_ID = "09d9ea69"
+KERNEL_MODULE_CONTAINERS.append(
+    OsContainer(
+        name="slm60-kernel-module-devel",
+        pretty_name="SUSE Linux Micro 6.0 Kernel module development",
+        logo_url="https://opensource.suse.com/bci/SLE_BCI_logomark_green.svg",
+        os_version=OsVersion.SLE16_0,
+        supported_until=_SUPPORTED_UNTIL_SLE.get(os_version),
+        is_latest=OsVersion.SLE16_0 in CAN_BE_LATEST_OS_VERSION,
```

Comment:
nit: not sure if this is a good idea as this will most likely make this container image `latest` once sle 16 is released, despite nothing changing for this container. It doesn't _really_ make a difference, but I guess there's no danger hardcoding this value?

Reply:
hardcoded to `True`